### PR TITLE
Made changes on Product Page

### PR DESCRIPTION
**Pull request by MinicMilica**
Made a few changes on Product Page, remove few things and fixed CSS on few things.

### CODE REVIEW

File: src/components/GMaps/GMaps.jsx [hunk 1]
```diff
@@ -2,40 +2,90 @@ import {
...
+}) {
+  const showOpeningHoursAndDays = (hours) => {
+    const daysWeek = ["Mon", "Tu", "We", "Th", "Fr", "Sa", "Su"];
+    let days = [];
+
+    days = hours?.places_placeopeninghours_days?.map(
+      (day) => day.places_day.number,
+    );
+
+    let dayss = [];
```
Comment (miloskapovic) [inline src/components/GMaps/GMaps.jsx:33]: name?

File: src/components/GMaps/GMaps.jsx [hunk 2]
```diff
@@ -2,40 +2,90 @@ import {
...
+    let daysss = "";
```
Comment (miloskapovic) [inline src/components/GMaps/GMaps.jsx:34]: name

File: src/components/GMaps/GMaps.jsx [hunk 3]
```diff
@@ -2,40 +2,90 @@ import {
...
+        } else {
+          dayss = days.map((day) => daysWeek[day]);
+          daysss = `${dayss.map((d) => d)}`.split(",");
+        }
+      });
+    } else {
+      dayss = days.map((day) => daysWeek[day]);
+      daysss = `${dayss.map((d) => d)}`;
+    }
+    let hourss = "";
```
Comment (miloskapovic) [inline src/components/GMaps/GMaps.jsx:49]: name

**Review by miloskapovic — CHANGES_REQUESTED** [review]

File: src/components/GMaps/GMaps.jsx (hunk 3 above)
Comment (MinicMilica) [reply src/components/GMaps/GMaps.jsx:49]: Done

**Review by miloskapovic — APPROVED** [review]

**Pull request merged**